src/abap/pretty_printer.ts

```diff
@@ -9,8 +10,13 @@ import {Position} from "../position";
 
 export interface IndentationOptions {
   alignTryCatch?: boolean;
+  globalClassSkipFirst?: boolean;
 }
 
+const classIsGlobal = (t: any) => {
```

Comment:
Maybe ... but which type ? If I understood well the children may contain at least Expressions and Statements. Can fix but tell me which type to use

Reply:
`statement` is a StatementNode, getChildren returns INode[], find is an array function applied to each item, so the input for classIsGlobal is `INode` ? (this is difficult without the editor to help)

Reply:
actually try using `statement.findFirstExpression(Expressions.ClassGlobal)` instead, it will traverse the full subtree, making it safe for future refactoring

Reply:
Good point with `findFirstExpression`, thanks. Fixed